.gitlab-ci.yml

```diff
@@ -6,58 +6,21 @@ stages:
   - build
   - release
 
-# Fallback to support `includes`
-cache:
-  key: $CI_COMMIT_REF_SLUG
-  paths:
-    - node_modules/
-    - lib/
-
-.shared:
-  artifacts:
-    paths:
-      - lib/
-      - package.json
-      - CHANGELOG.md
-    expire_in: 30 days
-  cache:
-    key: $CI_COMMIT_REF_SLUG
-    paths:
-      - node_modules/
-      - lib/
-
-build_npm_package:
-  stage: build
-  extends: .shared
-  except:
-    refs:
-      - schedules
-      - tags
-    variables:
-      - $CI_COMMIT_MESSAGE =~ /\[ci-release\]/
-  script:
-    - yarn build
-  cache:
-    policy: pull
-
 release:
```

Comment:
i'm sure there's a good reason but could you highlight what's keeping us from just using our shared `release.yml` from the project templates?

Reply:
added a comment :)

Reply:
i see, is there no way to extend from the common `release` and then just override the `variables` with the github ones?

Reply:
In theory yes. The problem though is that we'll end up with two `release` jobs: the included one and the custom one that extends from the former. That would result in a double-release and a race-condition where the 2nd `git push` would fail due to conflicts.

If you have an idea how to solve this, I'm all ears

Reply:
Understood, but it would only work when we move `.release` to `shared.yml`.

If we leave that in `release.yml`, all our existing projects would have to extend from that `.release`. Moving the extension to the individual `.gitlab-ci.yml`s is a step in the wrong direction imo. If we extend inside the same file where `.release` resides, we end up with two release jobs.

Reply:
ok, feels like it's getting too complicated to be DRY here. let's just leave it as is for now ✅